root/scripts/talker/gambler.txt

```diff
@@ -3166,8 +3166,10 @@ Rule EllisStoryInterruptGambler
 Response EllisStoryInterruptGenericGambler
 {
 	scene "scenes/Gambler/EllisInterrupt12.vcd"  //Ohhh, not again...
+	scene "scenes/Gambler/EllisInterrupt07.vcd" odds 5 //Hey Ellis, why don't you tell us about the time you SHUT UP?
```

Comment:
Will be more likely assigning this to c1m3 to reflect Nick's snobby attitude early on.